plugins/async-node/core/src/index.test.ts

```diff
@@ -114,6 +127,90 @@ test("should return current node view when the resolved node is undefined", asyn
   await asyncNodeTest(undefined);
 });
 
+test("should handle the promise using .then", async () => {
```

Comment:
Can you update the test name to something a bit more accurate to the new approach? Maybe `can handle multiple updates through callback mechanism`?

Reply:
done